src/tribler-core/tribler_core/components/restapi.py

```diff
@@ -109,12 +113,7 @@ def report_callback(text_long, sentry_event):
 
         CoreExceptionHandler.report_callback = report_callback
 
-        # We provide the REST API only after the essential endpoints (events, state and shutdown) and
-        # the exception handler were initialized
-        self.rest_manager = rest_manager
-
     async def shutdown(self):
```

Comment:
**nit**:  it doesn't relate to the PR, but it seems that `await super().shutdown()` is missed

Reply:
Fixed